picard/ui/coverartbox.py

```diff
@@ -28,6 +28,36 @@
 from picard.util import encode_filename, imageinfo
 
 
+class LRUCache(dict):
```

Comment:
Since this can be used by other modules too, mind moving this to a new utils.cache file?